rkt gc panics somewhere during network teardown

```
$ sudo ./build-rkt-0.16.0+git/bin/rkt --insecure-options=image run --interactive docker://busybox && sudo ./build-rkt-0.16.0+git/bin/rkt gc --grace-period=0
image: using image from file /home/kv/projects/coreos/rkt/build-rkt-0.16.0+git/bin/stage1-coreos.aci
image: remote fetching from URL "docker://busybox"
Downloading sha256:eeee0535bf3: [==============================] 676 KB/676 KB 
Downloading sha256:a3ed95caeb0: [==============================] 32 B/32 B
networking: loading networks from /etc/rkt/net.d
networking: loading network default with type ptp
/ # <CTRL+D>
gc: moving pod "56def87d-96c5-492f-8ceb-d6978604f9e8" to garbage
Garbage collecting pod "56def87d-96c5-492f-8ceb-d6978604f9e8"
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x8 pc=0x47efac]

goroutine 1 [running, locked to thread]:
github.com/coreos/rkt/networking.(*podEnv).teardownNets(0xc8200184e0, 0xc8200a6ae0, 0x1, 0x1)
	/home/kv/projects/coreos/rkt/build-rkt-0.16.0+git/gopath/src/github.com/coreos/rkt/networking/podenv.go:138 +0x14c
github.com/coreos/rkt/networking.(*Networking).Teardown(0xc8200184e0, 0xc82000ffe0, 0x6)
	/home/kv/projects/coreos/rkt/build-rkt-0.16.0+git/gopath/src/github.com/coreos/rkt/networking/networking.go:205 +0x18f
main.gcNetworking(0xc82000ff70, 0x0, 0x0)
	/home/kv/projects/coreos/rkt/build-rkt-0.16.0+git/gopath/src/github.com/coreos/rkt/stage1/gc/gc.go:79 +0x445
main.main()
	/home/kv/projects/coreos/rkt/build-rkt-0.16.0+git/gopath/src/github.com/coreos/rkt/stage1/gc/gc.go:57 +0x25b

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/lib/golang/src/runtime/asm_amd64.s:1721 +0x1
gc: problem performing stage1 GC on "56def87d-96c5-492f-8ceb-d6978604f9e8": exit status 2
```

--- a/networking/networking.go
+++ b/networking/networking.go
@@ -184,7 +184,10 @@ func (n *Networking) GetDefaultHostIP() (net.IP, error) {
 }
 
 // Teardown cleans up a produced Networking object.
-func (n *Networking) Teardown(flavor string) {
+func (n *Networking) Teardown(flavor string, debug bool) {
+
+	stderr = log.New(os.Stderr, "networking", debug)
+
 	// Teardown everything in reverse order of setup.
 	// This should be idempotent -- be tolerant of missing stuff
 

--- a/stage1/gc/gc.go
+++ b/stage1/gc/gc.go
@@ -76,7 +76,7 @@ func gcNetworking(podID *types.UUID) error {
 	n, err := networking.Load(".", podID)
 	switch {
 	case err == nil:
-		n.Teardown(flavor)
+		n.Teardown(flavor, debug)
 	case os.IsNotExist(err):
 		// probably ran with --net=host
 	default:

--- a/stage1/init/init.go
+++ b/stage1/init/init.go
@@ -516,7 +516,7 @@ func stage1() int {
 
 		if err = n.Save(); err != nil {
 			log.PrintE("failed to save networking state", err)
-			n.Teardown(flavor)
+			n.Teardown(flavor, debug)
 			return 6
 		}
 

